### PR TITLE
RHDEVDOCS-6148: Content creation for init and sidecar containers

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.14

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6148

Link to docs preview:

[Repo Server properties](https://81147--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties.html#argo-repo-server-properties_argo-cd-cr-component-properties)

ArgoCD Controller Options and ArgoCD Server Options hyperlinks in the Additional Resources section of [Installing a user-defined Argo CD instance](https://81147--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance.html#gitops-argo-cd-installation_setting-up-argocd-instance)

SME review: @iam-veeramalla 
QE review: @varshab1210 
Peer review: @tmalove 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by varshab1210** [comment]
LGTM, thanks

File: modules/gitops-repo-server-properties.adoc [hunk 1]
```diff
@@ -23,6 +23,8 @@ The following properties are available for configuring the Repo server component
 |`execTimeout` | `180` |Execution timeout in seconds for rendering tools (for example Helm or Kustomize).
 |`env` | `__<empty>__` |Environment to set for the repository server workloads.
 |`replicas` | `__<empty>__` |The number of replicas for the Argo CD Repo server. Must be greater than or equal to `0`.
+|`initContainers` | `__<empty>__` |The number of `init` containers in the ArgoCD Application Controller component. This field is optional.
+|`SidecarContainers` | `__<empty>__` |The number of `sidecar` containers in the ArgoCD Application Controller component. This field is optional.
```
Comment (varshab1210) [inline modules/gitops-repo-server-properties.adoc:27]:
```suggestion
|`sidecarContainers` | `__<empty>__` |The number of `sidecar` containers in the ArgoCD Application Controller component. This field is optional.
```

Comment (Dhruv-Soni11) [reply modules/gitops-repo-server-properties.adoc]: Fixed.

**Comment by tmalove** [comment]
/remove-label peer-review-in-progress
/label peer-review-done

@Dhruv-Soni11  Remember to squash your commits before requesting a merge review. Thanks!

**Pull request merged**